index.html

```diff
@@ -352,17 +352,64 @@ <h4>Script Type</h4>
-                <aside class="example">If a movie contains dialogue in English and Hebrew, the <a>Original Language Dialogue List</a> will contain some <a>Script Events</a> in English and some in Hebrew. None of the <a>Script Events</a> is translated.</aside>
+                <li>
+                  <dfn>Original Language Dialogue List</dfn>:
+                  When the <code>ttm:role</code> value is <code>x-DUBBING_ORIGINAL_DIALOGUE_LIST</code>,
+                  the document represents a literal transcription of the dialogue and on-screen text in their original spoken/written language(s).
+                <aside class="note"><a>Text</a> objects in the Original Language Dialogue List have their <a>Text Language Source</a> set to <a>Original</a>.</aside>
+                <aside class="example">If a movie contains dialogue in English and Hebrew,
+                  the <a>Original Language Dialogue List</a> will contain some <a>Script Events</a> in English and some in Hebrew,
+                  all marked as <a>Original</a> <a>Text Language Source</a>.
+                  None of the <a>Script Events</a> is marked as <a>Translation</a>.</aside>
                 </li>
-                <li><dfn>Translated Dialogue List</dfn>: When the <code>ttm:role</code> value is <code>x-DUBBING_TRANSLATED_DIALOGUE_LIST</code>, the document represents a translation of the dialogue or on-screen text in a common language. It is meant to be adapted to produce a <a>Pre-Recording Dub Script</a>. 
-                <aside class="note">A script can contain additional <a>Script Events</a> from the <a>Original Language Dialogue List</a> for context, to assist in the adaption process.</aside>
-                <aside class="example">If a movie contains dialogue in English and Hebrew, the French <a>Translated Dialogue List</a> will contain at least the translation in French of all <a>Script Events</a>. It may still retain text content in Hebrew and English to assist further processing.</aside>
+                <li>
+                  <dfn>Translated Dialogue List</dfn>:
+                  When the <code>ttm:role</code> value is <code>x-DUBBING_TRANSLATED_DIALOGUE_LIST</code>,
+                  the document represents a translation of the dialogue or on-screen text in a common language.
+                  It is meant to be adapted to produce a <a>Pre-Recording Dub Script</a>. 
+                <aside class="note">Translated <a>Script Events</a> contain <a>Text</a> objects whose
+                  <a>Text Language Source</a> is set to <a>Translation</a>.
+                  They can also contain <a>Original</a> language <a>Text</a>.</aside>
+                <aside class="note">A <a>Script Event</a> can contain <a>Translation</a> <a>Text</a> objects in more than one language, for context,
```

Comment:
I think this should say:
```
A <a>Script Event</a> in this type of script has a <a>Translation</a> <a>Text</a> object in the target recording language but it may contain an <a>Original</a> <a>Text</a> object, for context, to assist in the adaption process.</aside>
```

Reply:
Would we need to define "target recording language"?